driver/docker/version.go

```diff
@@ -0,0 +1,182 @@
+package docker
+
+import (
+	"github.com/Masterminds/semver/v3"
+)
+
+type mobyBuildkitVersion struct {
+	MobyVersionConstraint string
+	BuildkitVersion       string
+}
+
+// https://gist.github.com/crazy-max/780cb6878c37cb79ec3f7699706cf83f
```

Comment:
What's the link for this for? Is this the source of truth for this struct?

Reply:
To keep track of my investigation to recover the right BuildKit version so we known how these constraints have been made.